packages/manager/apps/telecom/src/app/telecom/pack/migration/meeting/pack-migration-meeting.controller.js

```diff
@@ -45,59 +47,84 @@ export default class TelecomPackMigrationMeetingCtrl {
           this.process.selectedOffer.address =
             eligibility.result.endpoint.address;
           this.process.selectedOffer.productCode = productCode;
-          return this.ovhApiConnectivityEligibilitySearch
-            .v6()
-            .searchMeetings(this.$scope, {
+          const installationType =
+            DICTIONNARY[this.process.selectedOffer.selectedPto];
+          this.searchMeetings(
+            eligibilityReference,
+            productCode,
+            installationType,
+          );
+        }
+        return null;
+      })
+      .catch((error) => {
+        this.tucToast.error(
+          `${this.$translate.instant(
+            'telecom_pack_migration_meeting_error',
+          )} ${get(error, 'data.message', '')}`,
+        );
+      })
+      .finally(() => {
+        this.loading = false;
+      });
+  }
+
+  searchMeetings(eligibilityReference, productCode, installationType) {
+    this.loading = true;
+    return this.PackMigrationMeetingService.searchMeetings(
+      eligibilityReference,
+      productCode,
+      installationType,
+    )
+      .then(({ data }) => {
+        if (data.status === 'pending') {
```

Comment:
after do https://github.com/ovh/manager/pull/16068#discussion_r2002890949

```suggestion
      .then(({ status, result }) => {
        if (status === 'pending') {
```
And 'pending' could be move in constant file